addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1341,6 +1345,7 @@ export class Wysiwyg extends Component {
                             }
                         }
                     });
+                    this.odooEditor.observerActive();
```

Comment:
Can you add a name parameter to make sure we don't "unlock" another unactive (add on both calls) ?
E.g. `fieldChanges` ?